test/cookbooks/verify-octo/attributes/default.rb

```diff
@@ -25,3 +25,5 @@
 # Tentacle test configuration
 default['verify-octo']['tentacle']['version'] = '3.1.3'
 default['verify-octo']['tentacle']['checksum'] = 'c5b4cd6ceec977137d93711ca0ede1dde79ae30da277d70e8d70e6d148860bec'
+default['verify-octo']['tentacle']['instance'] = 'Tentacle'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.